Makefile

```diff
@@ -50,9 +62,9 @@ $(OUT)/cpu_instrs.h: tests/cpu_instrs.gb tests/rom2h.c
 	$(Q)$(CC) -o $(OUT)/rom2h tests/rom2h.c
 	@$(OUT)/rom2h
 
-$(OUT)/bench: $(OUT)/cpu_instrs.h prof.h cpu.c bench.c gameboy.h
+$(OUT)/bench: $(GBIT_OBJS)
```

Comment:
Append `(OUT)/cpu_instrs.h prof.h cpu.c bench.c gameboy.h` after `$(GBIT_OBJS)`. Thus, we can track the dependency. Later, we can simplify them.